packages/terra-filter-view/src/_Frame.module.scss

```diff
@@ -0,0 +1,181 @@
+@import '~terra-mixins';
+
+:local {
+  .select-parent {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  .select {
+    background-color: var(--terra-filter-background-color, #fff);
+    color: var(--terra-filter-color, #1c1f21);
+    display: inline-flex;
+    font-size: var(--terra-filter-font-size, 1.143rem);
+    line-height: var(--terra-filter-line-height, 1.5);
+    position: relative;
+  }
+
+  .display {
+    overflow: hidden; // IE 10
+  }
+
+  .button {
+    border: 0.071em solid #dedfe0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0.25em;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0.25em;
+    display: inline-block;
+    flex: 0 0 auto;
+    margin: 0;
+
+    &:hover {
+      background-color: #b9bbbc;
+      border-color: #c8cacb;
+    }
+
+    &:active {
+      background-color: #6f7477;
+      border-color: #52585b;
+      color: #fff;
+    }
+  }
+
+  .clear-button {
+    color: var(terra-filter-clear-button-color, #b9bbbc);
```

Comment:
The variable name needs a double dash in front of it, e.g. `--terra-filter-clear-button-color`